proxy_map.go

```diff
@@ -1097,49 +1152,60 @@ func (as attributeSet) Attrs() []string {
 	return attrs
 }
 
+// MapEntryListenerConfig contains configuration for a map entry listener.
 type MapEntryListenerConfig struct {
 	Predicate    predicate.Predicate
 	Key          interface{}
 	flags        int32
 	IncludeValue bool
 }
 
+// NotifyEntryAdded enables receiving an entry event when an entry is added.
 func (c *MapEntryListenerConfig) NotifyEntryAdded(enable bool) {
 	flagsSetOrClear(&c.flags, int32(EntryAdded), enable)
 }
 
+// NotifyEntryRemoved enables receiving an entry event when an entry is removed.
 func (c *MapEntryListenerConfig) NotifyEntryRemoved(enable bool) {
 	flagsSetOrClear(&c.flags, int32(EntryRemoved), enable)
 }
 
+// NotifyEntryUpdated enables receiving an entry event when an entry is updated.
 func (c *MapEntryListenerConfig) NotifyEntryUpdated(enable bool) {
 	flagsSetOrClear(&c.flags, int32(EntryUpdated), enable)
 }
 
+// NotifyEntryEvicted enables receiving an entry event when an entry is evicted.
 func (c *MapEntryListenerConfig) NotifyEntryEvicted(enable bool) {
 	flagsSetOrClear(&c.flags, int32(EntryEvicted), enable)
 }
 
+// NotifyEntryExpired enables receiving an entry event when an entry is expired.
 func (c *MapEntryListenerConfig) NotifyEntryExpired(enable bool) {
 	flagsSetOrClear(&c.flags, int32(EntryExpired), enable)
 }
 
+// NotifyEntryAllEvicted enables receiving an entry event when an entry is evicted.
```

Comment:
Is it doing same thing with `NotifyEntryEvicted` method? Explenations are the same

Reply:
Thanks, fixed at https://github.com/hazelcast/hazelcast-go-client/pull/592/commits/db2c5ee518e06486f197a91fd252d442fc1de894